rapid7_insightvm/komand_rapid7_insightvm/actions/update_site_included_targets/action.py

```diff
@@ -25,7 +25,7 @@ def run(self, params={}):
             current_scope = resource_helper.resource_request(endpoint=endpoint,
                                                              method='get')
             self.logger.info(f"Appending to current list of included targets")
-            scope.extend(current_scope['addresses'])
+            scope.extend(current_scope.get('addresses'),[])
```

Comment:
I think you want this here: 
```suggestion
            scope.extend(current_scope.get('addresses', []))
```